src/main/web_src/src/components/ui/form/fieldArray/dollyFieldArray.less

```diff
@@ -68,6 +68,70 @@
 	}
 }
 
+.dfa-blokk-org-odd {
```

Comment:
Disse 2 er helt like uten fargene? Noen mulighet for å merge?

Reply:
Uff, ja, denne sleit jeg sikkelig med... Prøvde meg på nytt nå, og lagde en felles klasse for header og content, så ble det ihvertfall mye mindre duplikat-kode 😊